src/routes/arraysRouter.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const arraysController = require('../controllers/arraysController');
```

Comment:
maybe try destructuring with your controller functions so const { arrays_getNthElement, etc } = require('')

Nothing wrong with taking the whole object in and using dot notation to access. Destructuing would just save you writing the controller name out each time

Reply:
I think this was from how I was following along with the example MDN provides, for some reason I couldn't get my head around separating the routes and controllers, doing it this way seemed clearer at the time. Looking at now though I think destructuring is the way.